Portal/vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/server/dao/mysql/SSHData.java

```diff
@@ -148,16 +116,17 @@ public void updateSSH(SSH ssh) throws DAOException {
         try {
             PreparedStatement ps = connection.prepareStatement("UPDATE "
                     + "VIPSSHAccounts "
-                    + "SET sshUser=?, sshHost=?, sshDir=?, sshPort=?, auth_failed='0' "
+                    + "SET sshUser=?, sshHost=?,transfertType=?, sshDir=?, sshPort=?, auth_failed='0' "
                     + "WHERE email=? AND LFCDir=?");
 
             ps.setString(1, ssh.getUser());
             ps.setString(2, ssh.getHost());
-            ps.setString(3, ssh.getDirectory());
-            ps.setString(4, ssh.getPort());
-            ps.setString(5, ssh.getEmail());
+            ps.setString(3, ssh.getTransfertType());
+            ps.setString(4, ssh.getDirectory());
+            ps.setString(5, ssh.getPort());
+            ps.setString(6, ssh.getEmail());
             try {
-                ps.setString(6, DataManagerBusiness.generateLFCDir(ssh.getName(),ssh.getEmail()));
+                ps.setString(7, DataManagerBusiness.generateLFCDir(ssh.getName(),ssh.getEmail()));
             } catch (DataManagerException ex) {
                 logger.error(ex);
                 throw new DAOException(ex);
```

Comment:
Missing interface for:
* deleteFilesFromSource
* theEarliestNextSynchronistation
* numberSynchronizationFailed